<ui version="4.0" >
 <class>PrintDeviceDialogClass</class>
 <widget class="QDialog" name="PrintDeviceDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>450</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>WinMusik: Print Cover</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources.qrc" >
    <normaloff>:/icons/resources/printer32.png</normaloff>:/icons/resources/printer32.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="minimumSize" >
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../resources.qrc" >:/icons/resources/printer64.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>From device:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="fromDevice" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>377</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>to:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="toDevice" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="printColors" >
          <property name="text" >
           <string>print colors</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Use font:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="font" />
          </item>
          <item>
           <spacer name="horizontalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="printCoverButton" >
          <property name="text" >
           <string>print cover</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="printTracklistButton" >
          <property name="text" >
           <string>print tracklist</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="okButton" >
          <property name="text" >
           <string>print</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources.qrc" >
            <normaloff>:/icons/resources/button_ok.png</normaloff>:/icons/resources/button_ok.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>183</width>
            <height>29</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton" >
          <property name="text" >
           <string>cancel</string>
          </property>
          <property name="icon" >
           <iconset resource="../resources.qrc" >
            <normaloff>:/icons/resources/cancel.png</normaloff>:/icons/resources/cancel.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections/>
</ui>
